src/app/containers/redirect/redirect.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { CookieService } from 'ngx-cookie';
+import { AppUtils } from 'src/app/utils/app-utils';
+
+@Component({
+    selector: 'app-redirect',
+    template: ``
+  })
+
+export class RedirectComponent implements OnInit {
```

Comment:
probably this can be changed to directive rather than component if no template

Reply:
Agree but using component can give flexibility of showing loading icon